src/js/apps/forms/form/form-patient_app.js

```diff
@@ -45,9 +47,12 @@ export default App.extend({
     this.form = form;
     this.isReadOnly = this.form.isReadOnly();
 
+    const formWidgets = this.form.getWidgets();
+    const widgets = Radio.request('entities', 'widgets:collection', collectionOf(formWidgets, 'id'));
```

Comment:
I know the `collectionOf` stuff was probably in the patient sidebar app, but I think we should move this line into the `getWidgets` such that the function returns the instantiated widgets collection.  The stuff on the form options and org settings is a little un-standard so there's not clear patterns but I think it makes sense to work with the widgets as a collection at the earliest possible opportunity.

Reply:
Makes sense to me. Get the widgets data in the correct format as soon as possible. Will update that for both `form_app.js` and `form-patient_app.js`.